### PR TITLE
Changes for ODSC Tutorial

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Please enter a clear and concise description of what your pull request does, and why
it is necessary. If your patch fixes an issue, please reference that issue here.

**Testing steps**
Enter steps to test your pull request. Give a clear and concise description of
what you expected to happen during testing.

**Logs**
```
If applicable, please paste the command line from your testing:
```

**Other information**
Any other information or context you would like to provide.

**Data tests (if applicable)**
If you added a new teacher, you will be asked to run
`python setup.py test -s tests.suites.datatests`. Please paste this log here.

### CODE REVIEW

File: parlai/core/params.py [hunk 1]
```diff
@@ -1000,6 +1000,83 @@ def parse_args(self, args=None, namespace=None, print_args=True):
 
         return self.opt
 
+    def _kwargs_to_strargs(self, **kwargs):
```
Comment (klshuster) [inline parlai/core/params.py:1003]: just for my understanding, this function takes a dict of args and converts to what you'd write on the command line?

File: parlai/utils/strings.py [hunk 2]
```diff
@@ -64,13 +64,18 @@ def uppercase(string: str) -> str:
...
     BLUE = '\033[1;94m'
-    BOLD_LIGHT_GRAY = '\033[1;37;40m'
-    LIGHT_GRAY = '\033[0;37;40m'
+    BOLD_LIGHT_GRAY = '\033[1m'
+    LIGHT_GRAY = '\033[0m'
     MAGENTA = '\033[0;95m'
-    HIGHLIGHT_RED = '\033[1;37;41m'
-    HIGHLIGHT_BLUE = '\033[1;37;44m'
+    HIGHLIGHT_RED = '\033[1;31m'
+    HIGHLIGHT_BLUE = '\033[0;34m'
```
Comment (klshuster) [inline parlai/utils/strings.py:78]: just curious what you're doing here

File: parlai/zoo/tutorial_transformer_generator/build.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Pretrained model allowing to get the same performances as in
+https://arxiv.org/abs/1905.01969.
```
Comment (klshuster) [inline parlai/zoo/tutorial_transformer_generator/build.py:8]: nit: think you linked the wrong paper? (this is polyencoder)

File: parlai/agents/local_human/local_human.py [hunk 4]
```diff
@@ -63,7 +63,12 @@ def observe(self, msg):
     def act(self):
         reply = Message()
         reply['id'] = self.getID()
-        reply_text = input(colorize("Enter Your Message:", 'field') + ' ')
+        try:
+            reply_text = input(colorize("Enter Your Message:", 'text') + ' ')
+        except EOFError:
```
[the remark this answers is not in the document]
Comment (stephenroller) [reply parlai/agents/local_human/local_human.py:68]: If you type ctrl-D as I do by habit constantly lol

File: parlai/core/params.py [hunk 5]
```diff
@@ -1000,6 +1000,83 @@ def parse_args(self, args=None, namespace=None, print_args=True):
...
+                string_args.append(last_option_string)
+                string_args.append(str(value))
+            elif isinstance(action, argparse._StoreAction) and action.nargs in '*+':
+                string_args.append(last_option_string)
+                string_args.extend([str(v) for v in value])
+            else:
+                raise TypeError(f"Don't know what to do with {action}")
+
+        self.add_extra_args(string_args)
+        kwname_to_action = {}
```
[the remark this answers is not in the document]
Comment (stephenroller) [reply parlai/core/params.py:1035]: yeah, first time around is BASICALLY just to call add_extra_args. I'll add a comment

File: parlai/core/params.py (hunk 1 above)
Comment (stephenroller) [reply parlai/core/params.py:1003]: yup

File: parlai/utils/strings.py (hunk 2 above)
Comment (stephenroller) [reply parlai/utils/strings.py:78]: getting rid of background colors

File: parlai/zoo/tutorial_transformer_generator/build.py (hunk 3 above)
Comment (stephenroller) [reply parlai/zoo/tutorial_transformer_generator/build.py:8]: yup

**Pull request merged**